server/fleet/datastore.go

```diff
@@ -1,33 +1,344 @@
 package fleet
 
+import "time"
+
+type CarveStore interface {
```

Comment:
`CarveStore` remains because it is also implemented by the S3 datastore, right?

Reply:
Right, we implement it for mysql and s3 and then use it as part of the service http://github.com/fleetdm/fleet/blob/4149a50742904b0af7a7799740626e8eaf703e64/server/service/service.go#L24-L24